src/main.rs

```diff
@@ -86,6 +86,7 @@ pub fn app(app_state: AppState) -> Router {
     let routes = Router::new()
         .route("/healthz", get(healthz))
         .route("/deploy-api-key", post(deploy_api_key))
+        .route("/delete-secret", post(delete_secret))
```

Comment:
If we wanted to conform to REST style we could use DELETE with the same route name as the POST. But doesn't really matter.